tests/ImageProcessorCore.Tests/Processors/ProcessorTestBase.cs

```diff
@@ -23,6 +23,7 @@ public abstract class ProcessorTestBase
         {
             //"TestImages/Formats/Jpg/Floorplan.jpeg", // Perf: Enable for local testing only
             "TestImages/Formats/Jpg/Calliphora.jpg",
+            "TestImages/Formats/Jpg/IMG_5084.JPG",
```

Comment:
Are the attached jpegs supposed to be black?

Also, can we find a smaller TIFF, It bloats the repo size if too many large images are included.